src/components/OcSearchBar.vue

```diff
@@ -158,20 +162,27 @@ export default {
       // please don't treat empty string the same as null...
       return this.value === null ? this.query : this.value
     },
-    $_ocSpinner_class() {
+    spinnerSize() {
       if (this.small) {
-        return "oc-spinner oc-spinner-xs"
+        return "xsmall"
       }
-      return "oc-spinner oc-spinner-m"
+      return "medium"
     },
-    $_ocSearchBar_input_class() {
+    inputClass() {
       const classes = ["oc-search-input"]
 
       this.icon && classes.push("oc-search-input-icon")
       !this.buttonHidden && classes.push("oc-search-input-button")
 
       return classes
     },
+    loadingAccessibleLabel() {
+      if (this.customLoadingAccessibleLabel) {
```

Comment:
Can be simplified to `this.customLoadingAccessibleLabel || this.$gettext("Loading results")`.